app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -480,6 +484,14 @@ open class BrowserActivity : DuckDuckGoActivity() {
             return
         }
 
+        val params = intent.getActivityParams(BrowserActivityWithParams::class.java)
+        val url = params?.url
+        val interstitialScreen = params?.interstitialScreen ?: true
+        if (url != null) {
+            intent.putExtra(EXTRA_TEXT, url)
+            intent.putExtra(LAUNCH_FROM_INTERSTITIAL_EXTRA, interstitialScreen)
```

Comment:
You don't need to do this, just use `AppBrowserNav.openInNewTab`

Reply:
Fixed [here](https://github.com/duckduckgo/Android/pull/6129/commits/250fd32813216cf603a2b60e6b4810abe1051966).